pages/about.vue

```diff
@@ -2,31 +2,31 @@
   <v-container grid-list-xs>
     <v-row id="about">
       <!-- First row-->
-      <v-row id="header" class="first-row mb-10 pa-12">
+      <v-row id="header" class="first-row" :class="pageSectionClass">
         <v-col cols="12" :md="7" :sm="12">
```

Comment:
In another branch I changed `default.vue` to not have left or right padding [details here](https://github.com/covid19risk/website/pull/35#issuecomment-623542677).

Could you add `class="px-12"` on any `v-col` that contains copy that does NOT have a colored background?  This will make sure the colored background goes across the full screen while also giving the non-colored background copy left and right padding.

Reply:
Can we add classes for this? Maybe we should use negative padding on the coloured backgrounds, since they're less common, and give them `:class=[pageSectionRow, fullWidthBackground]` or something? (I also think we should rename `pageSectionClass` to `pageSectionRow`